src/main/java/com/minecolonies/achievements/MineColoniesAchievementPage.java

```diff
@@ -10,17 +10,15 @@
  *
  * @since 0.2
  */
-public class MineColoniesAchievementPage extends AchievementPage
-{
+public class MineColoniesAchievementPage extends AchievementPage {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)